src/index.js

```diff
@@ -11,24 +11,9 @@ export {SEPARATOR} from './exporter';
 
 export default function(deckName) {
   const exporter = new Exporter(getDb(), getZip());
-  const { topDeckId, topModelId } = exporter;
 
-  exporter.db.run(getTemplate());
-
-  exporter
-    .updateInitialDecksWith({
-      name: deckName,
-      top_deck_id: topDeckId
-    })
-    .updateInitialModelsWith({
-      name: deckName,
-      css: getCssTemplate(),
-      did: topDeckId,
-      id: topModelId,
-    });
-
-  return ['addCard', 'addMedia', 'save'].reduce((prev, i) => {
-    prev[i] = exporter[i];
-    return prev;
-  }, {});
+  return exporter
```

Comment:
I'll do -- PR still in WIP ( I'll add mark about )

I just move code from one place to another without changing logic.